### DISCUSSION TITLE
Create an issue with GraphAPI v4

### DISCUSSION THREAD

**Discussion by wei** [community/community#21722] [New to GitHub]
<p>Hi can someone help me finding documentation for creating an issue with GraphAPI v4? I see the v3 version at <a href="https://developer.github.com/v3/issues/#create-an-issue" rel="nofollow noopener">https://developer.github.com/v3/issues/#create-an-issue</a> and just want to see the equivilant in v4.</p>
<p>Appreciate it!</p>
<p>wei</p>

**Comment by wei** [comment]
<p>I looked in mutations but was unable to find anything related to issues. : )</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/wei">@wei</a>,</p>
<p>It is not currently possible to create Issues through the GraphQL API. However, the GraphQL API is in active development and adding this ability is being discussed in the <a href="https://platform.github.community/t/schema-request-create-issues-labels-milestones-etc/5355/4" rel="nofollow noopener">GitHub Platform Community</a>.</p>
<p>Thanks!</p>

**Comment by happyshows** [comment]
<p>Any updates on this thread?</p>
<p>I saw CreateIssuePayload is listed on documentation site, but not sure how to use it.</p>
